controllers/lobby.controller.js

```diff
@@ -1,39 +1,90 @@
 const { lobbyService } = require('../services');
 
-const create = async (req, res) => {
+// create a lobby
+const createLobby = async (req, res) => {
     try {
-        const data = await lobbyService.create(req);
+        const data = await lobbyService.createLobby(req);
         res.send(data);
     } catch (error) {
         res.send(error);
     }
 };
 
-const join = async (req, res) => {
+// join lobby with lobby code
+const joinLobby = async (req, res) => {
     try {
-        const data = await lobbyService.join(req);
+        const data = await lobbyService.joinLobby(req);
         res.send(data);
     } catch (error) {
         res.send(error);
     }
 };
 
-const play = async (req, res) => {
+// play lobby
+const playLobby = async (req, res) => {
     try {
-        const data = await lobbyService.play(req);
+        const data = await lobbyService.playLobby(req);
         res.send(data);
     } catch (error) {
         res.send(error);
     }
 };
 
-const status = async (req, res) => {
+const statusLobby = async (req, res) => {
     try {
-        const data = await lobbyService.status(req);
+        const data = await lobbyService.statusLobby(req);
         res.send(data);
     } catch (error) {
         res.send(error);
     }
 };
 
-module.exports = { create,join,play,status };
\ No newline at end of file
+
+
+const startRound = async (req, res) => {
+    try {
+        const data = await lobbyService.startRound(req);
+        res.send(data);
+    } catch (error) {
+        res.send(error);
+    }
+};
+
+const answerQuestions = async (req, res) => {
+    try {
+        const data = await lobbyService.answerQuestions(req);
+        res.send(data);
+    } catch (error) {
+        res.send(error);
+    }
+};
+
+
+const commonQuestion = async (req, res) => {
+    try {
+        const data = await lobbyService.commonQuestion(req);
+        res.send(data);
+    } catch (error) {
+        res.send(error);
+    }
+};
+
+const votingQuestions = async (req, res) => {
+    try {
+        const data = await lobbyService.votingQuestions(req);
+        res.send(data);
+    } catch (error) {
+        res.send(error);
+    }
+};
+
+const roundWinnder = async (req, res) => {
+    try {
+        const data = await lobbyService.roundWinnder(req);
+        res.send(data);
+    } catch (error) {
+        res.send(error);
+    }
+};
+
+module.exports = { createLobby, joinLobby, playLobby, statusLobby, startRound, answerQuestions, commonQuestion,votingQuestions, roundWinnder };
```

Comment:
Spacing needs to be fixed. If you are not already, please use a formatter to ensure all code is consistently formatted.

Reply:
each export should be on a separate line.